cmake/cpack.cmake

```diff
@@ -0,0 +1,60 @@
+
+# Default CPack generators
+set(CPACK_GENERATOR TGZ STGZ)
+
+# Detect OS type, OS variant and target architecture
+if(UNIX)
+  if(APPLE)
+    set(OS_TYPE macos)
+  else()
+    set(OS_TYPE linux)
+    # Determine distribution
+    execute_process(COMMAND lsb_release -si
+        OUTPUT_VARIABLE LINUX_DISTRO
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+  endif()
+  # Determine architecture
+  execute_process(COMMAND uname -m
+    OUTPUT_VARIABLE MACHINE_ARCH
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+  )
+  # Set OS type and ARCH suffix
+  set(OS_TYPE_ARCH_SUFFIX ${OS_TYPE}-${MACHINE_ARCH})
+elseif(WIN32)
+  set(OS_TYPE win)
+  # Determine architecture
+  if(CMAKE_SIZEOF_VOID_P EQUAL 8)
+    set(MACHINE_ARCH 64)
+  else()
+    set(MACHINE_ARCH "")
+  endif()
+  # Set OS type and ARCH suffix
+  set(OS_TYPE_ARCH_SUFFIX ${OS_TYPE}${MACHINE_ARCH})
+endif()
+
+include(InstallRequiredSystemLibraries)
+# Package release version
+set(PACKAGE_RELEASE_VERSION 1)
+set(CPACK_INSTALL_CMAKE_PROJECTS "${CPACK_INSTALL_CMAKE_PROJECTS};.;libbot;ALL;/")
+
+set(CPACK_PACKAGE_DIRECTORY ${CMAKE_BINARY_DIR}/packages)
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "libbot2")
+set(CPACK_PACKAGE_VENDOR "Kitware Inc.")
+set(CPACK_PACKAGE_DESCRIPTION_FILE "${CMAKE_CURRENT_SOURCE_DIR}/README")
+set(CPACK_RESOURCE_FILE_LICENSE "${CMAKE_CURRENT_SOURCE_DIR}/LICENSE")
+set(CPACK_PACKAGE_VERSION_MAJOR "1")
+set(CPACK_PACKAGE_VERSION_MINOR "0")
+set(CPACK_PACKAGE_VERSION_PATCH "0")
+set(CPACK_PACKAGE_VERSION ${CPACK_PACKAGE_VERSION_MAJOR}.${CPACK_PACKAGE_VERSION_MINOR}.${CPACK_PACKAGE_VERSION_PATCH})
+set(CPACK_PACKAGE_NAME "${PROJECT_NAME}")
+set(CPACK_PACKAGE_FILE_NAME ${CPACK_PACKAGE_NAME}_${CPACK_PACKAGE_VERSION}-${PACKAGE_RELEASE_VERSION}_${OS_TYPE_ARCH_SUFFIX})
+set(CPACK_STRIP_FILES TRUE)
+set(CPACK_SOURCE_STRIP_FILES FALSE)
+
+# Debian specific
+set(CPACK_DEBIAN_PACKAGE_RELEASE ${PACKAGE_RELEASE_VERSION})
+#set(CPACK_DEBIAN_PACKAGE_SHLIBDEPS "lcm (>=1.3.95-1)")
+set(CPACK_DEBIAN_PACKAGE_DEPENDS "lcm (>=1.3.95), libglu1-mesa, freeglut3")
```

Comment:
I have the list something closer to
```
freeglut3
lcm (>=1.3.95)
libc6
libgcc1 
libglib2.0-0
libglu1-mesa
libgtk2.0-0
libjpeg8
libpng12-0
libstdc++6 
python
python-gtk2
python-numpy
python-scipy
zlib1g
```

Reply:
@jamiesnape Indeed. I added a lot of dependencies.